webpack/assets/javascripts/react_app/components/common/forms/Button.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 
-export default ({ className, onClick, children, disabled = false }) => {
+export default ({ className, onClick, children, type = 'button', disabled = false }) => {
```

Comment:
👍 as not defaulting to `button` can cause a lot of problem in forms for cancel buttons, etc.

Reply:
also the name would be very confusing ;)